e2e_tests/tests/api_utils.py

```diff
@@ -46,7 +46,7 @@ def create_test_user(
     """
     session = admin_session()
     user = user or bindings.v1User(username=get_random_string(), admin=False, active=True)
-    password = get_random_string()
+    password = get_random_string() + "aA1"
```

Comment:
To the extent this is a useful change, the reason for it is a little unclear to users of `create_test_user`. Could you please add a comment and/or a docstring?

Reply:
Thank you. I think these e2e tests are better as a result of your changes, here.